modules/registry-refactoring/src/main/java/org/apache/airavata/registry/core/utils/QueryConstants.java

```diff
@@ -37,4 +37,26 @@
             "WHERE BQ.groupResourceProfileId LIKE : " + DBConstants.GroupResourceProfile.GROUP_RESOURCE_PROFILE_ID;
     String FIND_ALL_GROUP_COMPUTE_RESOURCE_POLICY = "SELECT CR FROM "+ ComputeResourcePolicyEntity.class.getSimpleName() + " CR " +
             "WHERE CR.groupResourceProfileId LIKE : " + DBConstants.GroupResourceProfile.GROUP_RESOURCE_PROFILE_ID;
+
+    String GET_USER_RESOURCE_PROFILE = "SELECT DISTINCT URP FROM " + UserResourceProfileEntity.class.getSimpleName() + " URP " +
```

Comment:
GET_USER_RESOURCE_PROFILE, GET_USER_COMPUTE_RESOURCE_PREFERENCE and GET_USER_STORAGE_PREFERENCE I don't think are needed if using the `get()` repository method as mentioned above.